src/main/java/org/sagebionetworks/bridge/models/schedules2/Schedule2.java

```diff
@@ -132,4 +159,10 @@ public long getVersion() {
     public void setVersion(long version) {
         this.version = version;
     }
+    @JsonIgnore
+    public Map<String, ActivityEventUpdateType> getStudyBurstsUpdateMap() {
+        return getStudyBursts().stream()
+                .filter(b -> b != null && b.getIdentifier() != null && b.getUpdateType() != null)
```

Comment:
Was this just an artifact of testing?

Reply:
Removed this entirely...it was only used in the validator, where we were already collecting the set of IDs.